src/daemon/rpmostreed-transaction.c

```diff
@@ -28,6 +28,7 @@
 
 struct _RpmostreedTransactionPrivate {
   GDBusMethodInvocation *invocation;
+  gboolean active;
```

Comment:
Bikeshed: maybe `done`, or `executed`, or `finished`? `active` could be misleading given that there are other things in this context that could be said to be "active".

Reply:
Done...though I should note that the `active` terminology already existed.